tools/otci/tests/test_otci.py

```diff
@@ -402,7 +402,19 @@ def _test_otci_multi_nodes(self, leader, commissioner, child1, child2):
         self.assertEqual('router', commissioner.get_state())
 
         for dst_ip in leader.get_ipaddrs():
-            commissioner.ping(dst_ip, size=10, count=1, interval=2, hoplimit=3)
+            statistics = commissioner.ping(dst_ip, size=10, count=10, interval=2, hoplimit=3)
+            self.assertTrue('transmitted_packets' in statistics)
+            self.assertEqual(statistics['transmitted_packets'], 10)
+            self.assertTrue('received_packets' in statistics)
+            self.assertEqual(statistics['received_packets'], 10)
+            self.assertTrue('packet_loss' in statistics)
+            self.assertAlmostEqual(statistics['packet_loss'], 0.0, delta=1e-9)
+            self.assertTrue('round_trip_time' in statistics)
+            rtt = statistics['round_trip_time']
+            self.assertTrue('min' in rtt)
+            self.assertTrue('avg' in rtt)
+            self.assertTrue('max' in rtt)
```

Comment:
```suggestion
```
not necessary because the next statement already guarantee that these three fields must exist in rtt.